radae_demo_tx.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+RADAE_PATH=$1
+RADAE_VENV=$2
+
+# Current RADAE scripts seem to require being executed from
+# the RADAE folder.
+cd $RADAE_PATH
+
+# The below does the following:
+#     * For each block of OTA audio from freedv-gui:
+#         * Convert the audio into IQ data via zero-padding.
+#         * Pass the IQ data into the RADAE encoder
+#         * Send the resulting 8 kHz audio back to freedv-gui.
+export PATH=$RADAE_VENV/bin:$PATH
+$RADAE_PATH/build/src/lpcnet_demo -features - - | $RADAE_VENV/bin/python3 $RADAE_PATH/radae_tx.py $RADAE_PATH/model19_check3/checkpoints/checkpoint_epoch_100.pth --auxdata | python3 $RADAE_PATH/f32toint16.py --real --scale 16383
```

Comment:
Looks :+1: But as above - see if the streaming works when you run the script form the cmd line.

Reply:
Yeah, they work on the command line. It looks like there's significant latency before they even start encoding or decoding, mainly while PyTorch is setting itself up (hence why I was having the TX problem). This might be able to be worked around by e.g. not constantly killing and restarting the TX process.